src/app.js

```diff
@@ -149,8 +152,8 @@ const telemetryConfig = {
 telemetry.init(telemetryConfig)
 
 // function to update the config
-function updateConfig (configString) {
-  configUtil.setConfig('CHANNEL_FILTER_QUERY_STRING', configString)
+function updateConfig (name, configString) {
```

Comment:
why is this function needed? can't the caller directly call configUtil.setConfig?

Reply:
Done